tests/tools/oecert/host/host.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include "../../../../common/sgx/endorsements.h"
 
+#define OECERT_ENCLAVE_FILENAME "enc/oecert_enc"
```

Comment:
why down a level instead of in the same directory?

Reply:
Previous command is in form of `host/oecert enc/oecert_enc OPTION`. I found the same routine in `tests/attestation_plugin` and `samples/attestation` that place the enclave image file under an enclave subdirectory.

Reply:
Prefer same directory too